README.md

```diff
@@ -245,23 +245,23 @@ pip3 install -r requirements-test.txt
 
 To run all unit tests, use:
 ```
-python3 -m unittest discover -p "test*.py"
+DATAPROFILER_SEED=0 python3 -m unittest discover -p "test*.py"
```

Comment:
doesn't this need a `;` separating it?

Reply:
Not necessary, but you can. That technically creates a seperate execution

https://unix.stackexchange.com/questions/159489/is-there-a-difference-between-and-and